charts/stable/photoprism/Chart.yaml

```diff
@@ -21,7 +21,11 @@ dependencies:
   - name: common
     repository: https://library-charts.k8s-at-home.com
     version: 4.3.0
+  - name: mariadb
+    version: 10.2.0
+    repository: https://charts.bitnami.com/bitnami
+    condition: mariadb.enabled
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: Upgraded `common` chart dependency to version `4.3.0`.
+      description: Added MariaDB database support
```

Comment:
Don't forget to add an entry for the updated application version 🙂

Reply:
Done